src/ontology/config/robot_report_external_content.txt

```diff
@@ -0,0 +1,8 @@
+ERROR	file:tmp/mondo/src/sparql/qc/general/qc-syn-source-not-xref.sparql
+ERROR	invalid_xref
+ERROR	duplicate_scoped_synonym
+ERROR	file:tmp/mondo/src/sparql/qc/general/qc-duplicate-exact-synonym-no-abbrev.sparql
+ERROR	file:tmp/mondo/src/sparql/qc/mondo/qc-proxy-merges.sparql
+ERROR	file:tmp/mondo/src/sparql/qc/general/qc-related-exact-synonym.sparql
```

Comment:
Consider removing this check since it looks like it is covered by the default ROBOT check of `duplicate_scoped_synonym` already included in this file.

Reply:
Agree!